pkg/scheduler/eventhandlers.go

```diff
@@ -292,6 +294,20 @@ func vmPodShouldSleep(pod *v1.Pod) bool {
 	return pod.Status.Phase == v1.PodNoSchedule
 }
 
+func podIsNetworkReady(pod *v1.Pod) bool {
+	if v, ok := pod.Annotations[arktosv1.NetworkReadiness]; ok {
+		v = strings.TrimSpace(v)
```

Comment:
is it a good idea to treat empty string or blanks as the "true" condition?

Reply:
as absence of such annotation is treated as true, empty/blank valued annotation treated as true is reasonable.